src/platform/site-wide/user-nav/components/SearchHelpSignIn.jsx

```diff
@@ -62,16 +65,16 @@ class SearchHelpSignIn extends Component {
       <div className="sign-in-links">
         {!isSubdomain && (
           <button className="sign-in-link" onClick={this.handleSignInSignUp}>
-            Sign in
+            {this.props.customText ? this.props.customText : 'Sign in'}
```

Comment:
Do we need to get an approval from the VSP team for changes outside the dhp-connected-devices app?

Reply:
@yonidoronpeters I don't believe so. But I'll double check, thanks for that

Reply:
@theodur This is what we are referring to in slack

Reply:
Any changes to files outside your team's application will require approval from the [codeowners](https://github.com/department-of-veterans-affairs/vets-website/blob/master/.github/CODEOWNERS#L23). In this case that would be @department-of-veterans-affairs/vsp-identity